src/main/java/jp/co/future/uroborosql/store/NioSqlManagerImpl.java

```diff
@@ -715,21 +769,28 @@ public Path getPath() {
 		private String getSqlBody() {
 			if (sqlBody == null) {
 				Path path = getPath();
-				if (Files.notExists(path)) {
+				URL url = Thread.currentThread().getContextClassLoader().getResource(getResourcePath(path));
+				if (url == null) {
```

Comment:
@HidekiSugimoto189 

With this fix, an exception will be thrown when placing the SQL file in a physical folder other than the classpath.  
Branch like the getLastModifiedTime method.

Reply:
@shout-star  
Thanks. fix it.